video/out/opengl/context_win.c

```diff
@@ -96,6 +96,7 @@ static bool create_dc(struct ra_ctx *ctx)
 
     pfd.iPixelType = PFD_TYPE_RGBA;
     pfd.cColorBits = 24;
+    pfd.cAlphaBits = 8;
```

Comment:
couldn't this cause less optional formats to be picked?
I would put this behind a check